### DISCUSSION TITLE
How Can I Handle Taskfile Updates

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [go-task/task#542] [Q&A]
In my Task scripts, one of the tasks updates a submodule with all new Taskfiles that are included in the main Taskfile. How can I stop the current execution of Task and restart over so that the new Taskfiles are loaded? I'm trying to avoid having Task running inside of Task.

**Comment by andreynering** [comment]
Hi @ProfessorManhattan,

That not possible. Update your submodules in one call and call your tasks on another one.
